File: sdkwork-ai-sdk-csharp/Models/RunResponse.cs
using System;
using System.Collections.Generic;
using System.Text.Json.Serialization;

namespace Ai.Models
{
    public class RunResponse
    {
        public string? Id { get; set; }
        public string? Object { get; set; }
        public string? Status { get; set; }
        public string? Model { get; set; }
        public string? Instructions { get; set; }
        public List<object>? Tools { get; set; }
        public Dictionary<string, string>? Metadata { get; set; }
        public Usage? Usage { get; set; }
        public double? Temperature { get; set; }
        public int? CreatedAt { get; set; }
        public string? ThreadId { get; set; }
        public string? AssistantId { get; set; }
        public RequiredAction? RequiredAction { get; set; }
        public LastError? LastError { get; set; }
        public int? ExpiresAt { get; set; }
        public int? StartedAt { get; set; }
        public int? CancelledAt { get; set; }
        public int? FailedAt { get; set; }
        public int? CompletedAt { get; set; }
        public Dictionary<string, object>? ToolResources { get; set; }
        public double? TopP { get; set; }
        public int? MaxPromptTokens { get; set; }
        public int? MaxCompletionTokens { get; set; }
        public object? TruncationStrategy { get; set; }
    }
}

File: sdkwork-ai-sdk-go/types/run_response.go
package types

// 运行响应
type RunResponse struct {
	Id string `json:"id"`
	Object string `json:"object"`
	Status string `json:"status"`
	Model string `json:"model"`
	Instructions string `json:"instructions"`
	Tools []interface{} `json:"tools"`
	Metadata map[string]string `json:"metadata"`
	Usage Usage `json:"usage"`
	Temperature float64 `json:"temperature"`
	CreatedAt int `json:"created_at"`
	ThreadId string `json:"thread_id"`
	AssistantId string `json:"assistant_id"`
	RequiredAction RequiredAction `json:"required_action"`
	LastError LastError `json:"last_error"`
	ExpiresAt int `json:"expires_at"`
	StartedAt int `json:"started_at"`
	CancelledAt int `json:"cancelled_at"`
	FailedAt int `json:"failed_at"`
	CompletedAt int `json:"completed_at"`
	ToolResources map[string]interface{} `json:"tool_resources"`
	TopP float64 `json:"top_p"`
	MaxPromptTokens int `json:"max_prompt_tokens"`
	MaxCompletionTokens int `json:"max_completion_tokens"`
	TruncationStrategy interface{} `json:"truncation_strategy"`
}
